components/script/dom/xr.rs

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
Should this file be called `xrsystem.rs` now?

Reply:
Yes, and you will need to change the reference in components/script/dom/mod.rs as well.